wuzz.go

```diff
@@ -155,7 +155,7 @@ func (a *App) Layout(g *gocui.Gui) error {
 			return err
 		}
 		setViewDefaults(v)
-		v.Title = "URL - press ctrl+r to send request"
+		v.Title = "URL - press ctrl+r to send request (F2)"
```

Comment:
Maybe it would be better right after the `URL` string. The current could also mean <kbd>F2</kbd> is for sending request.